contracts/Initializable.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity 0.5.16;
+
+contract Initializable {
+
+    address public _initializationManager;
```

Comment:
why _initializationManager and not initializationManager?

Reply:
Openzeppelin do this together with the getter.
But I can't really think of a good reason, I'll change this

Reply:
Actually this should be private, so that derived contracts can only read this using the getter